src/routes/+layout.svelte

```diff
@@ -147,7 +147,7 @@
 					{/each}
 				</div>
 
-				<div class="flex flex-col items-center gap-[1.5vw] mt-[2.5vw]">
+				<div class="flex flex-col items-center md:gap-5 2xl:gap-9">
```

Comment:
Why using `2xl` ? We can use `md:gap-[vw]`?

Reply:
I copied your codes